build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/ClientConfigurationFields.java

```diff
@@ -35,6 +35,9 @@
     String PACKAGE_MANAGER_MODULE = "package.manager.module"; // Custom module name for the build-info
     String PIP_ENV_ACTIVATION = "pip.env.activation";
     String DOTNET_USE_DOTNET_CORE_CLI = "dotnet.use.dotnet.core.cli";
+    String DOCKER_IMAGE_TAG = "docker.image.tag";
+    String DOCKER_HOST = "docker.host";
+    String DOCKER_PROPERTIES = "docker.properties";
```

Comment:
What is String "docker.properties"? Isn't this just properties we attach to artifacts? If so, why can't we use the usual (generic) way fo attaching properties to artifacts?

Reply:
Removed.